android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/protocol/MobileWalletAdapterServer.java

```diff
@@ -348,8 +346,8 @@ public static class AuthorizationResult {
         @Nullable
         public final Uri walletUriBase;
 
-        @NonNull @Size(min = 1)
-        public final AuthorizedAccount[] accounts;
+        @NonNull
+        public final AuthorizedAccount account;
```

Comment:
Summarizing my understanding:

We're going from `AuthorizedAccount[]` to `AuthorizedAccount` because only the first item of the array was ever used anyways. Change it to a single account for clarity (until its actually implemented).

Reply:
in v1.x, this was never even an array, just a single public key and account label. the `AuthorizedAccount` struct was added in the 2.0 upgrade. This is reverting some of the original 2.0 changes to be single account tho for clarity, yes.